decidim-proposals/config/locales/en.yml

```diff
@@ -298,6 +298,7 @@ en:
           email_outro: You have received this notification because you are following %{author_nickname}. You can stop receiving notifications following the previous link.
           email_subject: New proposal "%{resource_title}" by %{author_nickname}
           notification_title: The <a href="%{resource_path}">%{resource_title}</a> proposal was published by <a href="%{author_path}">%{author_name} %{author_nickname}</a>.
+          notification_title_official: The <a href="%{resource_path}">%{resource_title}</a> official proposal was published.
```

Comment:
I tested this according to the bug description at 9279 and I noticed that when the user is following the participatory space the translation is missing:
![image](https://user-images.githubusercontent.com/864340/177959125-3cdd656b-b963-4524-b88f-d236e045a87b.png)

So I believe you also need to add this translation to the `proposal_published_for_space` section below this one.

Reply:
Fixed, thanks!